src/app/services/variable.service.ts

```diff
@@ -67,11 +67,8 @@ export class VariableService {
     Observable.combineLatest(
       this.countService.selectedConceptCountMapUpdated.asObservable(),
       this.treeNodeService.treeNodesUpdated.asObservable()
-    ).subscribe(res => {
-      const isTreeLoadingFinished = res[1];
-      if (isTreeLoadingFinished) {
```

Comment:
I don't understand why this condition was needed, also the `res[1]` from  `combineLatest` subscription was not always a tree ( depending on what was emitted as the latest, it was sometimes a result of the count map update)